packages/core/src/components/popover/popover.md

```diff
@@ -1,52 +1,114 @@
+
+export class PopoverExample extends React.Component<{}, {}> {
+    public render() {
+        // popover content gets no padding by default; add the "pt-popover-content-sizing"
+        // class to the popover to set nice padding between its border and content,
+        // and a default width when inline.
+        return (
+            <Popover
+                interactionKind={PopoverInteractionKind.CLICK}
+                popoverClassName="pt-popover-content-sizing"
+                position={Position.RIGHT}
+            >
+                <Button intent={Intent.PRIMARY}>Popover target</Button>
+                <div>
+                    <h5>Popover title</h5>
+                    <p>...</p>
+                    <Button className="pt-popover-dismiss">Dismiss</Button>
+                </div>
+            </Popover>
+        );
+    }
+}
+```
+
+@### Position
+
+The `position` prop controls the Popover's position relative to the target. There are two attributes to consider:
+
+- Which <span class="docs-popover-position-label-side">__side__</span> of the target the popover should render on.
+- The popover's <span class="docs-popover-position-label-alignment">__alignment__</span> relative to the target.
+
+These two attributes can be expressed with a single value having the following structure:
+
+<pre class="docs-popover-position-value-code-block">
+    <span class="docs-popover-position-label-side">[SIDE]</span>_<span class="docs-popover-position-label-alignment">[ALIGNMENT]</span>
+</pre>
+
+The __@blueprintjs/core__ package exports a `Position` enumeration that contains the full set of supported side/alignment combinations.
+
+#### Example
+
+The following example shows all supported `Position`s and how each behaves in practice. Note that if <strong><code>_<span class="docs-popover-position-label-alignment">[ALIGNMENT]</span></code></strong> is ommitted, the popover will align to the __center__ of the target.
+
+@reactExample PopoverPositionExample
+
+#### Automatic positioning
+
+The `position` can also be set to the string literal `"auto"`, the default setting. In this mode, the Popover will continually re-position itself to the side with the most space available, adjusting its alignment intuitively as well. This is useful for guaranteeing that the Popover remains visible while scrolling within a parent container.
+
+@### Modifiers
 
 Modifiers are the tools through which you customize Popper.js's behavior. Popper.js defines several of its own modifiers to handle things such as flipping, preventing overflow from a boundary element, and positioning the arrow. `Popover` defines a few additional modifiers to support itself. You can even define your own modifiers, and customize the Popper.js defaults, through the `modifiers` prop. (Note: it is not currently possible to configure `Popover`'s modifiers through the `modifiers` prop, nor can you define your own with the same name.)
```

Comment:
below this, in a part you didn't touch, is a heading with poor phrasing:

```
Popover modifiers, _which cannot be used by you_
```

can you update it?

```
Popper.js modifiers which are ignored in Popover:
```

Reply:
language clarification: those "cannot be used" modifiers are custom ones that I wrote for `Popover` functionality, hence the distinction between "Popover/Popper.js modifiers."

Reply:
Went with: **Popper.js modifiers that `Popover` manages and that cannot be customized:**